ros2cli/ros2cli/node/direct.py

```diff
@@ -35,8 +35,12 @@ def timer_callback():
             args, 'node_name_suffix', '_%d' % os.getpid())
         start_parameter_services = getattr(
             args, 'start_parameter_services', False)
+
+        if not node_name:
```

Comment:
How about checking explicitly for `node_name is None`?

Reply:
Sounds good to me, `node_name is None` in 048791c